spinnaker-monitoring-daemon/spinnaker-monitoring/__main__.py

```diff
@@ -140,15 +137,17 @@ def disable_ssl_verification():
     # Handle target environment that doesn't support HTTPS verification
     ssl._create_default_https_context = _create_unverified_https_context
 
-def main():
+def main(config_search_path):
   """The main program sets up the commands then delegates to one of them."""
 
   disable_ssl_verification()
 
   all_command_handlers, parser = prepare_commands()
   opts = parser.parse_args()
   options = vars(opts)
-  options = util.merge_options_and_yaml_from_path(options, opts.config)
+  if options.get('config'):
+    config_search_path.append(options.get('config'))
```

Comment:
Are users actually specifying custom config locations?

Reply:
probably not, but it was already there.
I think it makes sense especially if not using halyard.